docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,99 @@
+"""
+Coriolis parameters over the UK
+===============================
+
+This example shows the coriolis parameters for the area encompassing the UK.
+
+It creates and plots a map showing the coriolis parameters, overlaid on top of
+the UK coastline. The code also creates a custom cube to store the data in and
+to plot it from.
+
+"""
+
+import cartopy.crs as ccrs
+
+import iris
+
```

Comment:
The blank line between Cartopy and Iris is (PEP8-)correct though. However, matplotlib and numpy should go in the same block as Cartopy.